R/data.R

```diff
@@ -131,3 +131,41 @@
 #' @keywords metadata
 #' @family metadata
 "atoxgr_criteria_ctcv5"
+
+#' Metadata Holding Grading Criteria for DAIDs
+#'
+#' @details
+#' This metadata has its origin in the ADLB Grading Spec Excel file which ships with `{admiral}`
+#' and can be accessed using `system.file("adlb_grading/adlb_grading_spec.xlsx", package = "admiral")`
+#' in sheet = "DAIDS".
+#' The dataset contained in there has the following columns:
+#' - `SOC`: variable to hold the SOC of the lab test criteria.
+#' - `TERM`: variable to hold the term describing the criteria applied to a particular lab test,
+#'   eg. 'Anemia' or 'INR Increased'. Note: the variable is case insensitive.
+#' - `SUBGROUP` : Description of sub-group of subjects were grading will be applied (i.e. >= 18 years)
+#' - `Grade 1`: Criteria defining lab value as Grade 1.
+#' - `Grade 2`: Criteria defining lab value as Grade 2.
+#' - `Grade 3`: Criteria defining lab value as Grade 3.
+#' - `Grade 4`: Criteria defining lab value as Grade 4.
+#' - `Grade 5`: Criteria defining lab value as Grade 5.
+#' - `Definition`: Holds the definition of the lab test abnormality.
+#' - `FILTER` : `admiral` code to apply the filter based on SUBGROUP column.
+#' - `GRADE_CRITERIA_CODE`: variable to hold code that creates grade based on defined criteria.
+#' - `SI_UNIT_CHECK`: variable to hold unit of particular lab test. Used to check against input data
+#'   if criteria is based on absolute values.
+#' - `VAR_CHECK`: List of variables required to implement lab grade criteria. Use to check against
+#'   input data.
+#' - `DIRECTION`: variable to hold the direction of the abnormality of a particular lab test
+#'   value. 'L' is for LOW values, 'H' is for HIGH values. Note: the variable is case insensitive.
+#' - `COMMENT`: Holds any information regarding rationale behind implementation of grading criteria.
+#'
+#' Note: Variables `SOC`, `TERM`, `SUBGROUP`, `Grade 1`, `Grade 2`,`Grade 3`,`Grade 4`,`Grade 5`, `Definition`
+#' are from the source document on DAIDS website defining the grading criteria.
+#' [**Division of AIDS (DAIDS) Table for Grading the Severity of Adult and Pediatric Adverse Events
```

Comment:
```suggestion
#' [Division of AIDS (DAIDS) Table for Grading the Severity of Adult and Pediatric Adverse Events](https://rsc.niaid.nih.gov/sites/default/files/daidsgradingcorrectedv21.pdf)
```

What about linking out here?

Reply:
Done